back/engines/commercial/nlp/app/services/nlp/api.rb

```diff
@@ -112,6 +112,30 @@ def text_network_analysis(tenant_id, project_id, locale, max_nodes: nil, min_deg
       response.parsed_response.dig('data', 'task_id')
     end
 
+    # Create a task to perform text-network analysis for a list of inputs.
+    #
+    # @param tenant_id [String]
+    # @param input_identifiers [Array<String>]
+    # @param max_nodes [Integer,nil]
+    # @param min_degree [Integer,nil]
+    # @return [Hash{String => String}] task identifier by language
+    def text_network_analysis_by_ids(tenant_id, input_identifiers, max_nodes: nil, min_degree: nil)
```

Comment:
Currently the methods are called:

- `text_network_analysis`
- `text_network_analysis_by_ids` 

I wonder if 

- `text_network_analysis_for_project`
- `text_nextwork_analysis_for_inputs` 

would be more intention-revealing, because it would mention the expected object type in both cases. What do you think?

Reply:
Good point! On the other hand, text network analysis is always applied to inputs, so in a sense it's always "for inputs". What changes is how the set of inputs is identified. Maybe, we could change it to:
- `text_network_analysis` -> `text_network_analysis_for_project` 
(or `project_text_network_analysis` to be consistent with `project_tag_suggestions` - or we could even just drop it).

- `text_network_analysis_by_ids` -> `text_network_analysis`. It could make sense since providing the list of identifiers explicitly is probably the simplest/most straightforward way of specifying the inputs (as long as we don't introduce a method that accepts the inputs (content) literally 😅).